Apps/W1/Shopify/app/src/Invoicing/Codeunits/ShpfyFulfillmentAPI.Codeunit.al

```diff
@@ -0,0 +1,53 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Codeunit Shpfy Fulfillment API (ID 30315).
+/// </summary>
+codeunit 30315 "Shpfy Fulfillment API"
+{
+    var
+        ShpfyJsonHelper: Codeunit "Shpfy Json Helper";
+        ShpfyCommunicationMgt: Codeunit "Shpfy Communication Mgt.";
+
+    /// <summary>
+    /// Fulfills shopify orders for each fulfillment id parsed from a completed draft order.
+    /// </summary>
+    /// <param name="JResponse">Json response from a completed draft order</param>
+    /// <param name="ShopCode">Shopify shop code to be used.</param>
+    internal procedure FulfillShopifyOrder(JResponse: JsonToken; ShopCode: Code[20])
```

Comment:
For consistency, separate ShopCode in to an individual SetShopCode procedure, and in that procedure set it to a Communication Mgt.